### PR TITLE
Create add.cpp

### PR DESCRIPTION
**Pull request by decentlearner**
adding numbers in cpp.

### CODE REVIEW

**Comment by CollectorContributor** [comment]
@decentlearner 
_

- Please Create an issue of it and mention it in this merge request, also rename the file as an Addition.CPP

_

**Review by CollectorContributor — CHANGES_REQUESTED** [review]
**It has compilation error,**

- on line number 8 , it should have return 0, instead of return o

**Comment by CollectorContributor** [comment]
@decentlearner am closing this merge request, please remove errors from code and create an issue of and resend the merge request

File: add.cpp [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#include<iostream>    // including iostream header file
+using namespace std;
+// Driver code
+int main(){
+  int a, b; // variable declaration of integer type.
```
Comment (vedant237) [inline add.cpp:5]: int a, b, sum ;

**Review by vedant237 — COMMENTED** [review]
Declared the data type of 'sum' variable

**Pull request closed without merging**